delivery_carrier_manual_weight/models/stock_picking.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    is_manual_weight = fields.Boolean(related="carrier_id.is_manual_weight")
+    weight_manual = fields.Float(string="Weight (Manual)", digits="Stock Weight")
+    shipping_weight_manual = fields.Float(string="Weight for Shipping (Manual)")
+
+    @api.depends(
+        "move_ids", "carrier_id", "carrier_id.is_manual_weight", "weight_manual"
```

Comment:
You cannot do that as if configuration changes, every picking will be recomputed.

```suggestion
        "move_ids", "carrier_id", "weight_manual"
```